packages/desktop/views/dashboard/governance/views/ProposalDetailsView.svelte

```diff
@@ -149,8 +159,10 @@
         openedQuestionIndex = questionIndex === openedQuestionIndex ? null : questionIndex
     }
 
-    function handleCancelClick(): void {
-        $governanceRouter.previous()
+    function onStopVotingClick(): void {
+        openPopup({
+            type: 'stopVoting',
+        })
     }
 
     function handleVoteClick(): void {
```

Comment:
Nice, this is coming into the coding conventions soon, so I'm happy you prefer this. Could you commit it in this handler as well

```suggestion
    function onVoteClick(): void {
```